cloud/src/test/scala/org/apache/spark/cloud/AzureInstantiationSuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
In the absence of any real tests, these do check that the transient stuff is picked up, and, at a surface level, there aren't fundamental differences between jackson versions.

I have a copy of the same tests in my downstream integration tests, where they are the first stop for detecting problems, the advantage being you don't need any credentials to run the tests —and they validate the spark-cloud exports, rather than the spark-cloud test CP. I'll cut them from here if you want